README.md

```diff
@@ -43,91 +39,85 @@ Following is the default configuration for `inpsyde/wp-translation-downloader` t
             "languages": [
                 "de_DE"
             ],
-            "directory": "public/wp-content/languages",
+            "directory": "public/wp-content/languages"
         }
     }
 }
 ```
 
-### Default configuration - own configuration file
-You can, if you have a lot of configurations, move the whole `wp-translation-downloader`-configuration to an own json-file and just provide the file path like this:
+### Configuration in custom file
+For better readability and portability, it is also possible use a different file which contains only the WP Translation Downloader configuration (everything that would go in the `extra.wp-translation-downloader` object).
 
-**composer.json**
+One use case could be to reuse the same configuration for many websites that are located in the same parent folder.
+
+For this it's necessary to use the configuration `"extra.wp-translation-downloader"` in `composer.json` to set the path of the custom file. The path must be relative to the folder containing `composer.json`:
 ```json
 {
-    "name": "vendor/my-package",
     "extra": {
-        "wp-translation-downloader": "./wp-translation-downloader.json"
+        "wp-translation-downloader": "../wp-translation-downloader.json"
```

Comment:
why using `../` ? The file is normally on the same level as the `composer.json`  which is mostly "outside" of the webroot.